### ISSUE TITLE
DASH output is much smaller than the mp4 input

### ISSUE THREAD

**Issue by parth2094** [google/shaka-packager#503]
## System info
**Operating System**: Ubuntu 14.04
**Shaka Packager Version**: 53aa775-release

## Issue and steps to reproduce the problem

**Packager Command**:
packager 'in=/media/test/data/inputAudio.mp4,stream=audio,init_segment=/media/audio/init.m4a,segment_template=/media/audio/$Number$.m4a' --segment_duration 10 --mpd_output /media/audio/master.mpd

Extra steps to reproduce the problem?
(1)
(2)

What is the expected result?
The input file is 10 hour long and 209 MB in size. The expected chunk files should sum up to roughly same number.

What happens instead?
The chunk files generated sum up to 14 MB which is roughly 40 minutes and the number of chunk files is 235 each 10 seconds which also sums upto 40 minutes.

<Please attach the input files or email to shaka-packager-issues@google.com.>
<Please provide any additional information below.>
In short, is there a limit on the input/output for shakapackager. The input mp4 file that I am using is 209 MB and the DASH output is 14 MB

**Comment by parth2094** [comment]
Hi Kqyang, thanks for the help. Your fix resolved the issue I was facing and after adding --generate_static_mpd I could generate chunks for the whole 10 hr file. Could you also explain the difference between a static mpd and dynamic?

**Comment by kqyang** [comment]
It is explained in [DASH spec](https://www.iso.org/standard/65274.html) in detail. Basically static mpd is for On-Demand content while dynamic mpd is for live services.

For dynamic mpd, Shaka Packager removes segments outside the live window automatically.

Does that answer your question?

I'll close this issue as your original issue has been resolved. Feel free to reply if you have other questions.
